packages/flutter_tools/lib/src/project.dart

```diff
@@ -806,47 +806,26 @@ $javaGradleCompatUrl
     if (result.version != AndroidEmbeddingVersion.v1) {
       return;
     }
-    globals.printStatus(
-'''
-━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
-Warning
-──────────────────────────────────────────────────────────────────────────────
-Your Flutter application is created using an older version of the Android
-embedding. It is being deprecated in favor of Android embedding v2. To migrate
-your project, follow the steps at:
-
-https://github.com/flutter/flutter/wiki/Upgrading-pre-1.12-Android-projects
-
-━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
-The detected reason was:
-
-  ${result.reason}
-━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
-''');
-    if (deprecationBehavior == DeprecationBehavior.ignore) {
-      BuildEvent('deprecated-v1-android-embedding-ignored', type: 'gradle', flutterUsage: globals.flutterUsage).send();
-      globals.analytics.send(
-        Event.flutterBuildInfo(
-        label: 'deprecated-v1-android-embedding-ignored',
-        buildType: 'gradle',
-      ));
-
-    } else { // DeprecationBehavior.exit
-      globals.analytics.send(
-        Event.flutterBuildInfo(
-        label: 'deprecated-v1-android-embedding-failed',
-        buildType: 'gradle',
-      ));
+    // The v1 android embedding has been deleted. TODO(gmackall): I think we
+    // will never get here (the imports will fail to resolve), so figure out a
+    // better way. This is just to simulate it so I can hunt down tests to
+    // modify or delete.
+    throwToolExit(
+      'Build failed due to use of deleted Android v1 embedding.',
```

Comment:
This change and the other one should NOT land. They are just to simulate the v1 embedding being deleted, to help me track down tests that need to be fixed or removed.

Reply:
Noting after merging for anyone who comes across this pr in search: leaving this exit in place (this is likely the first failure point we will hit for someone trying to build a v1 embedding app, and it is good to have a clear message)